### DISCUSSION TITLE
Management UI displays message rate as 0

### DISCUSSION THREAD

**Discussion by fallinsilence** [rabbitmq/rabbitmq-server#4679] [Questions]
![微信图片_20220429102123](https://user-images.githubusercontent.com/21984160/165876173-59b9fa84-b4d1-4e7c-b75c-44baffaea8b5.png)
Question is showing in this picture.What is the possible reason?Why message rate is 0?

**Comment by michaelklishin** [comment]
We cannot suggest anything specific from a single screenshot. We don't even know what version this is on. However, the selected time period is "last day".  Not every metric is [retained](https://rabbitmq.com/management.html#rates-mode) for hours. I recall
a few curious cases where default rate mode and retention period values affected how metrics for "last day" were displayed by the UI. Not a lot of details since it's a rarely used timeframe.

Use [Prometheus and Grafana](https://rabbitmq.com/prometheus.html) if you need data retention for multiple days.
